Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Use an official Python runtime as a parent image
+FROM ubuntu:20.04
```

Comment:
Why not 22.04? Or something more lightweight, like Alpine.

Reply:
Now it's Alpine.